stack.yaml

```diff
@@ -45,7 +45,8 @@ packages:
 
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
-# extra-deps:
+extra-deps:
+  - tasty-hedgehog-0.2.0.0@sha256:83a8b777fa472040979e44dba43c32441f55d5ddb9641a4d53deee4b0e09fa34
```

Comment:
I'm unhappy about this dependency being dropped from Stackage. That shouldn't block this update, but given this and all the other problems we have had, should we consider using something other than `tasty`?

Reply:
Not sure what the alternatives are. I'll create a card to look into this.